FuzzWrapper project is failing to build

# Environment

Windows build number: Version 10.0.18362.1082
Windows Terminal version (if applicable): commit f78687453cd63a2e92a146441fc482f1ec5ad598
Visual Studio version: 16.7.1 (community 2019)

# Steps to reproduce

1. Checkout latest code (commit f78687453cd63a2e92a146441fc482f1ec5ad598)
2. Open the _OpenConsole_ solution in Visual Studio.
3. Build the solution.

# Expected behavior

The entire solution should build successfully.

# Actual behavior

The _FuzzWrapper_ project is failing with multiple link errors.

```
3>ConTermParser.lib(OutputStateMachineEngine.obj) : error LNK2001: unresolved external symbol "class std::vector<class std::basic_string_view<wchar_t,struct std::char_traits<wchar_t> >,class std::allocator<class std::basic_string_view<wchar_t,struct std::char_traits<wchar_t> > > > __cdecl Microsoft::Console::Utils::SplitString(class std::basic_string_view<wchar_t,struct std::char_traits<wchar_t> >,wchar_t)" (?SplitString@Utils@Console@Microsoft@@YA?AV?$vector@V?$basic_string_view@_WU?$char_traits@_W@std@@@std@@V?$allocator@V?$basic_string_view@_WU?$char_traits@_W@std@@@std@@@2@@std@@V?$basic_string_view@_WU?$char_traits@_W@std@@@5@_W@Z)
3>ConTermParser.lib(OutputStateMachineEngine.obj) : error LNK2001: unresolved external symbol "bool __cdecl Microsoft::Console::Utils::StringToUint(class std::basic_string_view<wchar_t,struct std::char_traits<wchar_t> >,unsigned int &)" (?StringToUint@Utils@Console@Microsoft@@YA_NV?$basic_string_view@_WU?$char_traits@_W@std@@@std@@AEAI@Z)
3>ConTermParser.lib(OutputStateMachineEngine.obj) : error LNK2001: unresolved external symbol "class std::optional<struct til::color> __cdecl Microsoft::Console::Utils::ColorFromXTermColor(class std::basic_string_view<wchar_t,struct std::char_traits<wchar_t> >)" (?ColorFromXTermColor@Utils@Console@Microsoft@@YA?AV?$optional@Ucolor@til@@@std@@V?$basic_string_view@_WU?$char_traits@_W@std@@@5@@Z)
3>C:\...\terminal\bin\x64\Release\ConTerm.Parser.FuzzWrapper.exe : fatal error LNK1120: 3 unresolved externals
3>Done building project "FuzzWrapper.vcxproj" -- FAILED.
```

I think those functions were only recently added in PR #7578, and maybe it's just a lib reference that now needs to be added to the _FuzzWrapper_ project.

That said, I'm not sure if this is just my problem, because it doesn't seem to be failing in the CI builds.

--- a/src/terminal/parser/ft_fuzzwrapper/FuzzWrapper.vcxproj
+++ b/src/terminal/parser/ft_fuzzwrapper/FuzzWrapper.vcxproj
@@ -21,6 +21,9 @@
     <ClInclude Include="precomp.h" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\types\lib\types.vcxproj">
+      <Project>{18d09a24-8240-42d6-8cb6-236eee820263}</Project>
+    </ProjectReference>
     <ProjectReference Include="..\lib\parser.vcxproj">
       <Project>{3ae13314-1939-4dfa-9c14-38ca0834050c}</Project>
     </ProjectReference>

--- a/src/terminal/parser/ft_fuzzwrapper/FuzzWrapper.vcxproj.filters
+++ b/src/terminal/parser/ft_fuzzwrapper/FuzzWrapper.vcxproj.filters
@@ -33,4 +33,7 @@
       <Filter>Header Files</Filter>
     </ClInclude>
   </ItemGroup>
+  <ItemGroup>
+    <Natvis Include="$(SolutionDir)tools\ConsoleTypes.natvis" />
+  </ItemGroup>
 </Project>

